pkg/core/object/fmt.go

```diff
@@ -188,7 +188,7 @@ func validateSignature(obj *object.Object) error {
 	}
 
 	if err := icrypto.AuthenticateObject(*obj); err != nil {
-		return errors.New("invalid signature")
+		return fmt.Errorf("authenticate: %w", err)
```

Comment:
Not sure it's easily understandable, usually "authentication" is applied to users.

Reply:
we check the authenticity of object creation by its owner. Signature may be valid, but  Any suggestions?

Reply:
I'd say it's verification. But the difference is not so obvious here.

Reply:
i can leave this `verify signature:`

Reply:
I'd be somewhat ambiguous this way, we have a number of signatures and even some other checks in `AuthenticateObject`.

Reply:
then im leaving `authenticate` until a better option